scripts/ingests/utils.py

```diff
@@ -20,10 +20,18 @@
 import numpy as np
 
 warnings.filterwarnings("ignore", module='astroquery.simbad')
-logging.basicConfig(format='%(asctime)s %(levelname)s:%(message)s',
-                    datefmt='%m/%d/%Y %I:%M:%S %p')
 logger = logging.getLogger('SIMPLE')
 
+# Logger setup
+# This will stream all logger messages to the standard output and apply formatting for that
+logger.propagate = False  # prevents duplicated logging messages
+LOGFORMAT = logging.Formatter('%(asctime)s %(levelname)s:%(message)s', datefmt='%m/%d/%Y %I:%M:%S %p')
```

Comment:
```suggestion
LOGFORMAT = logging.Formatter('%(asctime)s %(levelname)s: %(message)s', datefmt='%m/%d/%Y %I:%M:%S%p')
```
Add space between level and message; remove space between time and pm/am; makes output look like:
```bash
10/06/2021 12:09:33PM INFO: Loading DB
10/06/2021 12:09:33PM DEBUG: DB cs is sqlite:///SIMPLE.db
10/06/2021 12:09:33PM WARNING: This is not right
10/06/2021 12:09:33PM ERROR: This is bad
10/06/2021 12:09:33PM CRITICAL: Everything is breaking
```